transfer_test.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"strconv"
+	"sync"
+	"testing"
+
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/commands/transferfiles"
+	"github.com/jfrog/jfrog-cli-core/v2/common/commands"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-cli/inttestutils"
+	"github.com/jfrog/jfrog-cli/utils/tests"
+	"github.com/jfrog/jfrog-client-go/utils"
+	clientTestUtils "github.com/jfrog/jfrog-client-go/utils/tests"
+	"github.com/stretchr/testify/assert"
+)
+
+var targetArtifactoryCli *tests.JfrogCli
+var targetServerDetails *config.ServerDetails
+
+func InitTransferTests() {
+	initArtifactoryCli()
+	cleanUpOldRepositories()
+	tests.AddTimestampToGlobalVars()
+	createRequiredRepos()
+	inttestutils.InstallDataTransferPlugin()
+	var creds string
+	creds, targetServerDetails = inttestutils.AuthenticateTarget()
+	targetArtifactoryCli = tests.NewJfrogCli(execMain, "jfrog rt", creds)
+	inttestutils.CreateTargetRepos(targetArtifactoryCli)
+	inttestutils.RefreshStorageInfoAndWait(serverDetails)
+}
+
+func CleanTransferTests() {
+	deleteCreatedRepos()
+	inttestutils.DeleteTargetRepos(targetArtifactoryCli)
+	cleanTestsHomeEnv()
+}
+
+func initTransferTest(t *testing.T) func() {
+	if !*tests.TestTransfer {
+		t.Skip("Skipping transfer test. To run transfer test add the '-test.transfer=true' option.")
+	}
+	oldHomeDir, newHomeDir := prepareHomeDir(t)
+
+	// Delete the target server if exist
+	config, err := commands.GetConfig("target", false)
```

Comment:
I suggest we give this server-id a uniquer name (`jfrog-cli-test-target` for example), and make it a constant.